katello_certs/hooks/boot/02-message-helpers.rb

```diff
@@ -21,21 +41,22 @@ def proxy_instructions_message(kafo)
 
   To finish the installation, follow these steps:
 
-  If you do not have the smartproxy registered to the Katello instance, then please do the following:
+  If you do not have the #{proxy_name} registered to the #{main_instance_name} instance, then please do the following:
 
   1. yum -y localinstall http://#{fqdn}/pub/katello-ca-consumer-latest.noarch.rpm
   2. subscription-manager register --org "<%= color('#{org}', :info) %>"
 
-  Once this is completed run the steps below to start the smartproxy installation:
```

Comment:
Any idea why it originally said `smartproxy`? Was that a typo? Was the file additionally processed somehow?

Reply:
It may be well possible nobody read it that closely or cared enough to create an issue.